store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -137,6 +141,31 @@ public void initialise(final Schema schema, final StoreProperties properties)
         TableUtils.ensureTableExists(this);
     }
 
+    @Override
+    public void validateSchemas() {
+        super.validateSchemas();
+
+        final ValidationResult validationResult = new ValidationResult();
+
+        getSchemaElements().entrySet().forEach(entrySet -> entrySet.getValue().getProperties().forEach(propertyName -> {
+            final Serialiser serialisation = entrySet.getValue().getPropertyTypeDef(propertyName).getSerialiser();
+
+            if (serialisation != null && !(serialisation instanceof ToBytesSerialiser)) {
```

Comment:
At this point no serialisers should be null. If they are then we should add an error to the validation result. E.g:

```java
if(null == serialisation) {
    validationResult.addError("Serialiser for " + propertyName + " in group " + entrySet.getKey() " is null. A default serialiser could not be found so you must specify your own serialiser class");
} else if(!(serialisation instanceof ToBytesSerialiser)) {
    validationResult.addError(
            String.format(SCHEMA_SERIALISER_S_FOR_PROPERTY_S_IN_THE_GROUP_S_IS_NOT_INSTANCE_OF_S,
                    serialisation.getClass().getName(),
                    propertyName,
                    entrySet.getKey(),
                    ToBytesSerialiser.class.getCanonicalName()));
}

```